src/longbow/annotate/command.py

```diff
@@ -88,27 +100,39 @@
     required=False,
     help="Minimum ccs-determined read quality for a read to be annotated.  CCS read quality range is [-1,1]."
 )
+@click.option(
+    '-f',
+    '--force',
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Force overwrite of the output files if they exist."
+)
 @click.argument("input-bam", default="-" if not sys.stdin.isatty() else None, type=click.File("rb"))
-def main(pbi, threads, output_bam, model, chunk, max_length, min_rq, input_bam):
+def main(pbi, threads, output_bam, model, chunk, min_length, max_length, min_rq, force, input_bam):
     """Annotate reads in a BAM file with segments from the model."""
 
     t_start = time.time()
 
     logger.info("Invoked via: longbow %s", " ".join(sys.argv[1:]))
 
+    # Check to see if the output files exist:
+    bam_utils.check_for_preexisting_files(output_bam, exist_ok=force)
+
     threads = mp.cpu_count() if threads <= 0 or threads > mp.cpu_count() else threads
     logger.info(f"Running with {threads} worker subprocess(es)")
 
     # Get our model:
     if LibraryModel.has_prebuilt_model(model):
-        logger.info(f"Using %s", LibraryModel.pre_configured_models[model]["description"])
         m = LibraryModel.build_pre_configured_model(model)
     else:
         logger.info(f"Loading model from json file: %s", model)
         m = LibraryModel.from_json_file(model)
+    logger.info(f"Using %s: %s", model, m.description)
```

Comment:
The stuff above is now a common enough task across commands that we should refactor it into a function that we stick into one of the utility files.

Reply:
👍 
Added issue #88 to track this.